api/v1alpha1/package_types.go

```diff
@@ -27,8 +27,9 @@ type PackageInfoTemplate struct {
 	// Name of the package to install
 	// +kubebuilder:validation:Required
 	Name string `json:"name"`
-	// Version of the package to install (optional)
-	Version string `json:"version,omitempty"`
+	// Version of the package to install
+	// +kubebuilder:validation:Required
```

Comment:
I think this can be omitted, since controller-gen infers this from the omission of the "omitempty" tag. (Please check if this is true)

Reply:
Yes you're right.